### DISCUSSION TITLE
Emails

### DISCUSSION THREAD

**Discussion by MikasaSucasa** [frc5567/2023-Robot#177] [Team Posts]
Hey guys, I want to send an email update to everyone but I seem to be having issues finding everyone's emails. Please respond to this with your NON-school email address. We have a lot to discuss and organize. @JooshO @OwenM14 @CooperTrout , I expect you three as the most senior members to be hunting people down for their contact information. Graduated seniors, attach your email if you want to be involved with the off-season work but this will not be required.

**Comment by JewelBoner** [comment]
jazzledjewel@gmail.com

**Comment by smarie7833** [comment]
samanthamarie7833@gmail.com

**Comment by ZZeller03** [comment]
zzeller03@gmail.com 
Zackary  Zeller

**Comment by ghost** [comment]
samuel.l.halliday@gmail.com

>
Sam Halliday

**Comment by MeganNichols2004** [comment]
mmnichols2004@yahoo.com

**Comment by RugBark** [comment]
mattruff78@gmail.com

**Comment by ABancroft548** [comment]
agreatbancroft@gmail.com

**Comment by JooshO** [comment]
joshoverbeek0@gmail.com

**Comment by CooperTrout** [comment]
coopertrout52@gmail.com

**Comment by matthewbogs711** [comment]
mattb3350@yahoo.com

**Comment by OwenM14** [comment]
owenmorrow@comcast.net

**Comment by MikasaSucasa** [comment]
There are a few more left out there. Feel free to give a friendly reminder. I will also be asking people to send me gmail accounts as we will be using a google sheet to track some progress and responsibilities. Good luck on exams!

**Comment by MikasaSucasa** [comment]
@IsaacLiimatta @CalebOverbeek @HannahStrimpel @MadreCoca

**Comment by pangoliff** [comment]
Caleb.overbeek@outlook.com

**Comment by HannahStrimpel** [comment]
h4rescue@gmail.com

-Hannah Strimpel
